apps/app/src/pages/search/[...params]/index.tsx

```diff
@@ -174,6 +274,16 @@ const SearchResults = () => {
 		}
 	}, [data, loadingPage])
 
+	const [stateInUS, setStateInUS] = useState<string>('')
+
+	useEffect(() => {
+		if (searchState.searchTerm?.slice(-3) == 'USA') {
+			setStateInUS(searchState.searchTerm?.split(', ')[1] || '')
+		} else {
+			setStateInUS('')
+		}
+	}, [searchState.searchTerm])
+
```

Comment:
We cannot rely on the result items to always be "[city], [state], [country]"
Some result items return neighborhood names, which would shift the position over by one. 
![image](https://github.com/weareinreach/InReach/assets/58997957/41b7bf57-e71f-4651-9773-b9eacbfa2e90)

Maybe we can create a tRPC API route that does a lookup using something like [this package](https://www.npmjs.com/package/latlongify) server-side (to avoid all this extra stuff ending up in a client js bundle) based off of the lat/long, which can be grabbed from the URL.

Reply:
Good point, although when I click on that the neighborhood is added to the front and the state still occurs formatted between the last two commas. Do you think checking for "USA" and the two chars between the last two commas would work?

Reply:
What happens when the language is set to another locale? Does the `USA` get returned as `EE`?

Reply:
Ooof yeah this doesn't work

Reply:
Moving the signal for the banner to the server as we discussed sounds good

Reply:
The API route has been created.